hack/util.sh

```diff
@@ -96,7 +96,7 @@ function start_os_server {
       --loglevel=4 \
       1>&2 2>"${BASETMPDIR}/openshift.log" &
   else
-    sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
+    env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
```

Comment:
Sudo is required in hack/util.sh for start_os_server for the all-in-one
because you need root.  We don't require running our hack scripts with
sudo.  So we need to continue to use sudo for running a full server.

On Mon, Aug 10, 2015 at 7:49 AM, David Eads <notifications@github.com>
wrote:

> In hack/util.sh
> <https://github.com/openshift/origin/pull/4072#discussion_r36623156>:
>
> > @@ -96,7 +96,7 @@ function start_os_server {
> >        --loglevel=4 \
> >        1>&2 2>"${BASETMPDIR}/openshift.log" &
> >    else
> > -    sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
> > +    env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
>
> sudo is required to be able to make services work correctly, so without
> privileged execution, the value of starting the server is severely limited.
> I thought that @smarterclayton <https://github.com/smarterclayton> had
> some bash insanity that got the job done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4072/files#r36623156>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift